EnrichStream/.run/EnrichStream.run.xml

```diff
@@ -0,0 +1,22 @@
+<component name="ProjectRunConfigurationManager">
+    <configuration name="EnrichStreamJob" type="Application" factoryName="Application" nameIsGenerated="true">
+        <envs>
+            <env name="SECURITY_PROTOCOL" value="SASL_PLAINTEXT" />
+            <env name="TOKEN_URL" value="http://localhost:1852/realms/local-development/protocol/openid-connect/token" />
+            <env name="KAFKA_CLIENT_ID" value="default-access" />
+            <env name="KAFKA_CLIENT_SECRET" value="default-access-secret" />
+            <env name="KAFKA_SCOPE" value="kafka" />
+            <env name="SCHEMA_REGISTRY_CLIENT_ID" value="default-access" />
+            <env name="SCHEMA_REGISTRY_CLIENT_SECRET" value="default-access-secret" />
+            <env name="SCHEMA_REGISTRY_SCOPE" value="schema-registry" />
+        </envs>
+        <option name="INCLUDE_PROVIDED_SCOPE" value="true" />
+        <option name="MAIN_CLASS_NAME" value="cheetah.example.job.EnrichStreamJob" />
+        <module name="${lowerCaseJobName}" />
+        <option name="PROGRAM_PARAMETERS" value="--kafka-bootstrap-servers localhost:9092 --kafka-group-id EnrichStreamJson-group-id --schema-registry-url http://localhost:8081/apis/ccompat/v7 --input-kafka-topic EnrichStreamEnrichTopic --input-kafka-topic EnrichStreamInputTopic --output-kafka-topic EnrichStreamOutputTopic" />
```

Comment:
Should the *-kafka-topic parameters match the parameters declared in the docker-compose? (Should postfix be included?)
In the docker compose file the input kafka topics are named input-kafka-topic-**enriching** and input-kafka-topic-**main-source**.

Reply:
I am fixing that on my branch